main_test.go

```diff
@@ -467,7 +467,7 @@ func TestBenchmarkLowRPSDuration(t *testing.T) {
 	start := time.Now()
 	main()
 	duration := time.Since(start)
-	assert.True(t, duration < 200*time.Millisecond, "Expected 100ms benchmark to complete within 200ms, took %v", duration)
+	assert.True(t, duration < 400*time.Millisecond, "Expected 100ms benchmark to complete within 400ms, took %v", duration)
```

Comment:
can you add a comment on why do we put 400 millisecond as a threshold here?

Reply:
This wasn't required, purged this line